### PR TITLE
EDSC-3828: Update edsc-timeline to use Node 18

### PR DESCRIPTION
**Pull request by eudoroolivares2016**
Using earthdata-download ci.yml for template
Using longterm support hydrogen

### CODE REVIEW

**Review by dpesall — APPROVED** [review]

**Review by daniel-zamora — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@ cypress-coverage
 cypress/screenshots
 cypress/videos
 cypress/fixtures/example.json
+.vscode
```
Comment (macrouch) [inline .gitignore:12]: Why was this added?

**Review by macrouch — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (eudoroolivares2016) [reply .gitignore:12]: That's just for me and my `.vscode` extensions. I can remove it if you like. I thought I had pushed that up to the other one but, I hadn't should definitely do it to all of them or none of them. I'm just trying to avoid accidently committing a change there
